docs/tutorials/00_amplitude_estimation.ipynb

```diff
@@ -0,0 +1,582 @@
+      "text/plain": [
+       "<Figure size 193.726x84.28 with 1 Axes>"
+      ]
+     },
+     "execution_count": 44,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "iae_circuit = iae.construct_circuit(problem, k=3)\n",
+    "iae_circuit.draw('mpl', style='iqx')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Maximum Likelihood Amplitude Estimation\n",
+    "\n",
+    "See [3]."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Estimate: 0.20002237175368104\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit.algorithms import MaximumLikelihoodAmplitudeEstimation\n",
+    "\n",
+    "mlae = MaximumLikelihoodAmplitudeEstimation(\n",
+    "    evaluation_schedule=3,  # log2 of the maximal Grover power\n",
+    "    quantum_instance=quantum_instance\n",
+    ")\n",
+    "mlae_result = mlae.estimate(problem)\n",
+    "\n",
+    "print('Estimate:', mlae_result.estimation)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Faster Amplitude Estimation\n",
+    "\n",
+    "See [4]."
```

Comment:
This is referencing an entry that does not seem to exist (there are just 3 refs below)

Reply:
Left as is, since I don't know what is the link @Cryoris  is referring to.

Reply:
Ah, it didn't save the final version. The reference is 
```
[4] Faster Amplitude Estimation. K. Nakaji (2020). https://arxiv.org/pdf/2003.02417.pdf
```

Reply:
added in c54145f